src/Bitwise.gren

```diff
@@ -11,6 +11,14 @@ module Bitwise exposing
 
 ## Bit Shifts
 
+Note: The shift distance is effectively limited to 31.
```

Comment:
Might want to say why that is.

Maybe add a sentence at the start of the module documentation that bitwise operations first turns a number into a 32-bit integer before performing their operation, then refer to that.

So at the top, before any `@docs`:

```
Bitwise operations only work on 32-bit integers. When an `Int` is passed to a bitwise operation, only the 32 least significant bits is used.
```

And here:

```
Note: Because bitwise operations only work on 32-bit integers, the shift distance is effectively limited to 31.
```